src/redux/actions/socialActions.js

```diff
@@ -0,0 +1,31 @@
+import jwtDecode from 'jwt-decode';
+import { GET_ERRORS, AUTH_SOCIAL } from './types/socialType';
+
+export const getCurrentUser = decoded => ({
+  type: AUTH_SOCIAL,
+  payload: decoded,
+});
+
+export const getError = error => ({
+  type: GET_ERRORS,
+  payload: error,
+});
+export const socialAuthPath = token => async (dispatch) => {
+  try {
+    localStorage.setItem('token', token);
+    const decoded = jwtDecode(token);
+    dispatch(getCurrentUser(decoded));
+  } catch (err) {
+    dispatch(getError(err));
+  }
+};
+
+export const getToken = (tokenString) => {
+  const startIndex = tokenString.indexOf('=') + 1;
+  const token = tokenString.slice(startIndex);
+  return token;
+};
+
+export const socialLink = (type) => {
+  window.location.href = `${process.env.API_BASE_URL}/auth/${type}`;
+};
```

Comment:
Your variable names seem a bit off, I'll just make few suggestions that show intent:
- GET_ERRORS => SOCIAL_LOGIN_FAILURE
- AUTH_SOCIAL => SOCIAL_LOGIN_SUCCESS
- getCurrentUser => setCurrentError
- socialAuthPath => authorizeSocialUser
- getError => (find something for this guy)

Makes sense? You can even come up with better names that clearly tells what each thing does